styles/desktop-styles.css

```diff
@@ -0,0 +1,153 @@
+.toolbar {
+  display: none;
+}
+
+.toolbar-desktop {
+  display: block;
+}
+
+.toolbar-desktop ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  height: 72px;
+  font-size: 15px;
+}
+
+.toolbar-desktop ul > * {
+  margin: 0 10px;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+a:visited {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+a:active {
+  text-decoration: none;
+}
+
+.toolbar-desktop ul > * a {
+  color: #344563;
+}
+
+header {
+  background-image: url(../media/Header-llustration-desktop.png);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-position: 0;
+  background-size: cover;
+}
+
+.headline {
+  max-width: 920px;
+  margin-top: 160px;
+}
+
+.info {
+  font-size: 20px;
+  padding: 0 75px;
+}
+
+.chevron-down {
+  margin-top: 82px;
+  margin-bottom: 392px;
+}
+
+.works {
+  margin-top: 110px;
+}
+
+.works-container {
+  max-width: 1155px;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, auto);
+  column-gap: 24px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 52px;
+}
+
+.indicator {
+  display: none;
+}
+
+.project-title {
+  margin: 0 16px;
+}
+
+.skills {
+  max-width: 1155px;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: auto;
+  column-gap: 24px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.skill-container {
+  margin: 0;
+  align-items: center;
+}
+
+.about-me {
+  background-image: url(../media/illustration-about\ me-1-desktop.png), url(../media/illustration-about-me-2-desktop.png);
+  margin-top: 192px;
+}
+
+.about-me-info {
+  max-width: 920px;
+  padding: 0 75px;
+  box-sizing: border-box;
+  margin-top: 24px;
+}
+
+.indicator2 {
+  display: none;
+}
+
+footer {
+  background-image: url(../media/illustration-2-contact-form-desktop.svg), url(../media/illustration-2-contact-form-desktop-2.svg), url(../media/illustration-2-contact-form-desktop-3.svg);
+  background-repeat: no-repeat, no-repeat, no-repeat;
+  background-position: 274px 529px, left 360px, right 223px;
+}
+
+.contact-form {
+  max-width: 1155px;
+  display: flex;
+  align-items: center;
+  padding-bottom: 225px;
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.contact-form-title {
+  max-width: 42%;
+  margin-right: 24px;
+  font-size: 30px;
+  text-align: left;
+  line-height: 52px;
+}
+
+.contact-form form {
+  width: 58%;
+  margin-top: 50px;
+  align-items: flex-start;
+}
+
+footer .social-media {
+  margin: 18px 0;
+}
+
+.myemail,
+.indicator2 {
+  display: none;
+}
```

Comment:
- [ ] You did well working on this task. However, creating a new CSS file for the desktop version is not a good practice. As per this project requirement, you are required to use media queries to handle the responsiveness for the desktop version. Please take a look at [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) to get more understanding about implementing media queries.